packages/@okta/vuepress-site/docs/reference/api/event-hooks/index.md

```diff
@@ -594,7 +594,7 @@ curl -v -X DELETE \
 }
 ```
 
-### channel Object
+### Channel Object
```

Comment:
Is the name of the object `channel` with a lowercase 'c'? If so, will it be misleading to change capitalization?

Reply:
Looking around elsewhere in our reference docs, we capitalize the way you have, so this looks like it's good to go.